app/views/play/menu/InventoryModal.js

```diff
@@ -452,6 +452,13 @@ module.exports = (InventoryModal = (function () {
         if (!slotEl.find('img').length) { if (unequippedSlot == null) { unequippedSlot = slotEl } }
       }
       slotEl = unequippedSlot != null ? unequippedSlot : firstSlot
+      const itemEl = this.unequipItemFromSlot(slotEl)
+      if ((itemEl != null ? itemEl.length : undefined)) {
+        itemEl.addClass('active')
+        const selectedSlotItemID = itemEl.data('item-id')
+        const item = this.items.get(selectedSlotItemID)
+        item.classes = _.without(item.classes, 'equipped')
+      }
```

Comment:
Not sure if these extra lines are needed? In my simple testing, just line 455 (calling `this.unequipItemFromSlot(slotEl)` worked.

```suggestion
      this.unequipItemFromSlot(slotEl)
```

Reply:
@nwinter Yes you're right, I tried to do it like the `unequipSelectedItem()` function but yes you're totally right `unequipItemFromSlot()` does it. Maybe `unequipSelectedItem()` could be refactored then